broker/src/main/java/org/apache/rocketmq/broker/config/v2/SubscriptionGroupManagerV2.java

```diff
@@ -72,7 +72,7 @@ private boolean loadSubscriptions() {
             while (iterator.isValid()) {
                 SubscriptionGroupConfig subscriptionGroupConfig = parseSubscription(iterator.key(), iterator.value());
                 if (null != subscriptionGroupConfig) {
-                    super.updateSubscriptionGroupConfigWithoutPersist(subscriptionGroupConfig);
+                    super.putSubscriptionGroupConfig(subscriptionGroupConfig);
```

Comment:
<img width="874" alt="image" src="https://github.com/user-attachments/assets/d06e71c2-b5ed-46a2-84c6-a3625596bb8f">
after update then print log?

Reply:
We are actually loading existing config from persistence kv store, not updating.

Given there may be a lot of items while reboot loading, log this may flood the log files unnecessarily.